docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/kubernetes.md

```diff
@@ -113,14 +138,315 @@ helm --namespace dagster-cloud upgrade agent \
     --values ./values.yaml
 ```
 
-## Troubleshooting tips
+### Use a secret to pull images
 
-You can see basic health information about your agent in the Dagster+ UI:
+The agent is responsible for managing the lifecycle of your code locations and will typically need to pull images after your CI/CD process builds them and pushes them to your registry. You can specify a secret the agent will use to authenticate to your image registry.
 
-![Screenshot of agent health information](/img/placeholder.svg)
 
-### View logs
+:::tip
+For cloud based Kubernetes such as AWS EKS, AKS, or GCP, you typically don't need an image pull secret. The role used by Kubernetes will have permission to access the registry. You can skip this configuration.
+:::
+
+
+First create the secret. This step will vary based on the registry you use, but for DockerHub:
+
+```
+kubectl create secret docker-registry regCred \
+  --docker-server=DOCKER_REGISTRY_SERVER \
+  --docker-username=DOCKER_USER \
+  --docker-password=DOCKER_PASSWORD \
+  --docker-email=DOCKER_EMAIL
+```
+
+Use Helm to configure the agent with the secret:
+
+```yaml file=values.yaml
+# values.yaml
+imagePullSecrets: [regCred]
+```
 
 ```shell
-kubectl --namespace dagster-cloud logs -l deployment=agent
+helm --namespace dagster-cloud upgrade agent \
+    dagster-cloud/dagster-cloud-agent \
+    --values ./values.yaml
+```
+
+### Don't send Dagster+ stdout and stderr
+
+By default, Dagster+ will capture stdout and stderr and present them to users in the runs page. You may not want to send Dagster+ these logs, in which case you should update the compute logs setting.
+
+<Tabs>
+
+<TabItem value="s3" label="If you use AWS S3 at your organization">
```

Comment:
```suggestion
<TabItem value="s3" label="Store logs in S3">
```

Reply:
Is this S3-specific only? e.g. GCS won't work?

Reply:
only s3